server/sonar-web/src/main/js/apps/projects/styles.css

```diff
@@ -136,10 +137,18 @@
   content: "";
 }
 
+.project-card-measure.pull-right {
```

Comment:
instead of doing this you could add both `pull-right` and `text-right` classes to the element